tests/lib/rules/no-implicit-globals.js

```diff
@@ -0,0 +1,233 @@
+/**
+ * @fileoverview Tests for no-implicit-globals rule.
+ * @author Joshua Peek
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require("../../../lib/rules/no-implicit-globals"),
+    RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var ruleTester = new RuleTester();
+ruleTester.run("no-implicit-globals", rule, {
+    valid: [
+        {
```

Comment:
Good catch. I didn't account for that.

Updated so any globals that are explicitly marked as writtable can still be assigned.